src/coresyftools/coresyftools/argument_parser.py

```diff
@@ -19,6 +19,8 @@ def __init__(self, manifest, logger=None):
         is_valid, errors = self._validate_manifest(self.manifest)
         if not is_valid:
             raise InvalidManifestException(errors)
+        self.identifiers = set()
+        self._configure_arg_parser()
 
     def _validate_manifest(self, manifest):
```

Comment:
The idea was to have an extendable semantic validation (i.e. beyond the syntactic validation of _cerebrus_). For example, if previous versions we needed to validate that `arguments` contained at least one element with type `data` and `output`. 
However this is not needed anymore, and I think that now the only field it can make sense extend the  validation `operation` that is the only one which is supposed to be independently interpreted by subclasses. 

The returned pair was, a boolean for the validation status and the list of found problems (it could be simply the list :stuck_out_tongue: ).

I removed this.